SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLStabilityControlsStatusSpec.m

```diff
@@ -0,0 +1,50 @@
+//
+//  SDLStabilityControlsStatusSpec.m
+//  SmartDeviceLink
+
+#import <Foundation/Foundation.h>
+#import <Nimble/Nimble.h>
+#import <Quick/Quick.h>
+
+#import "SDLRPCParameterNames.h"
+#import "SDLStabilityControlsStatus.h"
+#import "SDLVehicleDataStatus.h"
+
+QuickSpecBegin(SDLStabilityControlsStatusSpec)
+
+describe(@"getter/setter tests", ^ {
+    it(@"should set and get correctly", ^ {
+        SDLStabilityControlsStatus* testStruct = [[SDLStabilityControlsStatus alloc] init];
+        testStruct.escSystem = SDLVehicleDataStatusOn;
+        testStruct.trailerSwayControl = SDLVehicleDataStatusOn;
+        expect(testStruct.escSystem).to(equal(SDLVehicleDataStatusOn));
+        expect(testStruct.trailerSwayControl).to(equal(SDLVehicleDataStatusOn));
+    });
+
+    it(@"should get correctly when initialized with a dict", ^ {
+        NSDictionary* dict = @{SDLRPCParameterNameEscSystem:SDLVehicleDataStatusOn, SDLRPCParameterNameTrailerSwayControl:SDLVehicleDataStatusOn};
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+            SDLStabilityControlsStatus* testStruct = [[SDLStabilityControlsStatus alloc] initWithDictionary:dict];
+#pragma clang diagnostic pop
```

Comment:
Please update to initialize the dictionary the same way as other RPC tests

Reply:
also updated the test structure to be: describe-context-it.